dev/examples-to-tests.el

```diff
@@ -3,9 +3,12 @@
 (require 'dash-functional)
 
 (defun example-to-should (example)
-  (let ((actual (car example))
-        (expected (nth 2 example)))
-    `(should (equal ,actual ,expected))))
+  (-let [(actual sym expected) example]
```

Comment:
Not optimal, but probably ok I'd say. Besides, `-map` and `-partition` are already in there. :)